force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,32 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
+
+        String greetSF = 'Hello, Salesforce!';
         
-        // Get the length of the String        
+        // Get the length of the String       
+        Integer lengthOfGreeting = greetSF.length();
+
         // Get a substring of the String of the first 5 characters
+
+        String subGreeting  = greetSF.substring(0, 5);
+
         // Convert the String to lower case
+
+        String lowerGreeting = greetSF.toLowerCase();
         
         // Return the list with the string data
         // In the first position add the string length
         // In the second position add the substring
         // In the third position add the string in lower case
 
-        return null; // Replace null with the variable you used to store the result
+        List<String> greetingString = new List<String>();
+       
+        greetingString.add(String.valueOf(lengthOfGreeting));
+        greetingString.add(subGreeting);
+        greetingString.add(lowerGreeting);
```

Comment:
Try to do this in one line       
        greetingString.addAll(new List<String>{..........});